pennylane/debugging.py

```diff
@@ -221,18 +221,143 @@ def pldb_device_manager(device):
 
 
 def breakpoint():
-    """Launch the custom PennyLane debugger."""
+    """A function which freezes execution and launches the PennyLane debugger (PLDB).
+
+    This function marks a location in a quantum function (QNode). When it is encountered during
```

Comment:
quantum function is not a QNode, do we use them interchangeably?

Reply:
Nice catch, replaced with "quantum circuit"